src/pages/proposal.tsx

```diff
@@ -170,6 +170,7 @@ export default function ProposalPage() {
   const { votes, isLoadingVotes, reloadVotes } = useProposalVotes(proposal?.id)
   const { highQualityVotes, lowQualityVotes } = useMemo(() => getVoteSegmentation(votes), [votes])
   const { surveyTopics, isLoadingSurveyTopics } = useSurveyTopics(proposal?.id)
+  const [showResults, setShowResults] = useState(false)
```

Comment:
I think you can just add this to the proposalPageState, that is already being shared with the ProposalGovernanceSection

Reply:
Great!